src/runtime/pay-flow.js

```diff
@@ -82,6 +94,8 @@ export class PayStartFlow {
       'swg': {
         'publicationId': this.pageConfig_.getPublicationId(),
         'skuId': this.sku_,
+        'oldSkuId': this.oldSku_,
```

Comment:
Ideally we wouldn't send these keys at all if they are not relevant.

Reply:
done